lib/bolt/project_manager/module_migrator.rb

```diff
@@ -27,14 +28,21 @@ def migrate(project, configured_modulepath)
           true
         # Migrate modules from Puppetfile
         elsif File.exist?(puppetfile)
-          migrate_modules_from_puppetfile(config, puppetfile, managed_moduledir, modulepath)
+          migrate_modules_from_puppetfile(config, puppetfile, managed_moduledir, old_modulepath)
         # Migrate modules to updated modulepath
         else
-          consolidate_modules(modulepath)
+          consolidate_modules(old_modulepath)
           update_project_config([], config)
         end
       end
 
+      # Returns true if the project's modules can be migrated, false otherwise.
+      # To migrate a project's modules, the current project must use either
+      # the old or new default modulepath.
+      def migrateable?
```

Comment:
I don't see where we use this?

Reply:
Hah. Just a direction I abandoned and forgot to remove. 😅